blob/blob_fuzz_test.go

```diff
@@ -0,0 +1,92 @@
+package blob
+
+import (
+	"encoding/json"
+	"os"
+	"path/filepath"
+	"testing"
+
+	"github.com/celestiaorg/celestia-node/blob/blobtest"
+)
+
+func FuzzProofEqual(f *testing.F) {
+	if testing.Short() {
```

Comment:
Why do we need skip this?

Reply:
It is customary to avoid running such long running operations when a user explicitly requests for go test -short

Reply:
AFAIR fuzz test are run only with `-fuzz` flag, so user expects fuzzing to run.

Reply:
-fuzz when ran manually takes only one fuzzing function, when you put it on oss-fuzz and other continuous fuzzers there are a variety of conditions and specifications on the commandline, one of which is -short